ports/filament/cmake-build.patch

```diff
@@ -1016,12 +1016,15 @@ index 906f7441d..a10ed754d 100644
 -
 -endif()
 diff --git a/libs/viewer/CMakeLists.txt b/libs/viewer/CMakeLists.txt
-index 621bb2498..47f09cec2 100644
+index 621bb2498..7ee870af1 100644
 --- a/libs/viewer/CMakeLists.txt
 +++ b/libs/viewer/CMakeLists.txt
-@@ -49,12 +49,3 @@ endif()
+@@ -47,14 +47,5 @@ endif()
  # ==================================================================================================
- install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
+ # Installation
+ # ==================================================================================================
+-install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
++install(TARGETS ${TARGET} ARCHIVE DESTINATION lib)
```

Comment:
this was my OCD, filament viewer was getting copied into `lib/arm64` instead of just `lib`